mgmt_cli_test.py

```diff
@@ -391,6 +364,43 @@ def download_from_azure(self, node, source, destination):
         source = f"{source.replace('azure://', self.backup_azure_blob_service)}{self.backup_azure_blob_sas}"
         node.remoter.sudo(f"azcopy copy '{source}' '{destination}'")
 
+    @staticmethod
+    def create_s3_bucket(name: str, region: str) -> None:
```

Comment:
What are ACL's for these buckets? If open to all, let's make a note here about that.
I think we should protect against creating buckets in root, just to avoid spamming it with test stuff.
Same is for GCE.